docs/marko-testing-library/api.mdx

```diff
@@ -212,6 +215,32 @@ expect(emitted()).toMatchInlineSnapshot(`
   `)
 ```
 
+### `cleanup`
+
+Like the top level cleanup method cleanup method, this allows you to remove and
```

Comment:
```suggestion
Like the top level cleanup method, this allows you to remove and
```

Reply:
Good catch, thanks!

I went ahead and put the Marko Testing Library documentation through Grammarly to catch some issues like this and made a bit more cleanup / link fixes.